packages/generators/admin/component/templates/selectors.js.hbs

```diff
@@ -0,0 +1,17 @@
+import { createSelector } from 'reselect';
+import { initialState } from './reducer';
+{{#unless plugin "===" "admin"}}
+  import pluginId from '../../pluginId'
+{{/unless}}
+
+/**
+ * Direct selector to the {{name}} state domain
+ */
+
+export const select{{name}}Domain = state => state[`
```

Comment:
Can you add the `// eslint-disable-next-line import/prefer-default-export` comment so the lint passes?

Reply:
Thanks, I added local eslint rule disabling whenever it could not be automatically fixed by the new `lint` action (unfortunately I did not found a way to add this comments in the same step so it will have to stay in the templates).